squape/squishserver.py

```diff
@@ -0,0 +1,178 @@
+                f"\nstderr: {stderr}"
+            )
+
+    def addAUT(self, aut: str, path: str) -> None:
+        """Register an AUT
+
+        Args:
+            aut (str): the name of the executable
+            path (str): path to the executable folder
+        """
+        log(f"Registering {Path(path)/aut} AUT")
+        self._config_squishserver("addAUT", [aut, path])
+
+    def removeAUT(self, aut: str, path: str) -> None:
+        """Remove registered AUT
+
+        Args:
+            aut (str): the name of the executable
+            path (str): path to the executable folder
+        """
+        log(f"Removing registered {Path(path)/aut} AUT")
+        self._config_squishserver("removeAUT", [aut, path])
+
+    def addAppPath(self, path: str) -> None:
+        """Register an AUT path
+
+        Args:
+            path (str): the AUT path to register
+        """
+        log(f"Registering AUT path: {path}")
+        self._config_squishserver("addAppPath", [path])
+
+    def removeAppPath(self, path: str) -> None:
+        """Remove a registered AUT path
+
+        Args:
+            path (str): the path to the AUT
+        """
+        log(f"Removing registered AUT path: {path}")
+        self._config_squishserver("removeAppPath", [path])
+
+    def addAttachableAut(self, aut: str, port: int, host: str = "127.0.0.1") -> None:
+        """Register an attachable AUT
+
+        Args:
+            aut (str): the name of the attachable AUT
+            port (int): port of the machine where the attachable AUT
+                        is supposed to be running.
+            host (str, optional):   host of the machine where the attachable AUT
+                                    is supposed to be running.
+                                    Defaults to "127.0.0.1".
+        """
+        log(f"Registering an attachable AUT {aut} ({host}:{port})")
+        self._config_squishserver("addAttachableAUT", [aut, f"{host}:{port}"])
+
+    def removeAttachableAut(self, aut: str, port: int, host: str = "127.0.0.1") -> None:
+        """Remove registered attachable AUT
+
+        Args:
+            aut (str): the name of the attachable AUT
+            port (int): port of the machine where the attachable AUT
+                        is supposed to be running.
+            host (str, optional):   host of the machine where the attachable AUT
+                                    is supposed to be running.
+                                    Defaults to "127.0.0.1".
+        """
+        log(f"Removing registered attachable AUT {aut} ({host}:{port})")
+        self._config_squishserver("removeAttachableAUT", [aut, f"{host}:{port}"])
+
+    def attachToApplication(self, aut):
+        """
+        Attaches to an application with given name.
+
+        Args:
+            aut (str): the name of the attachable AUT
+        Returns:
+            ctx : Application Context
+        """
+        ctx = squish.attachToApplication(aut, self.host, self.port)
```

Comment:
I'd  add a log message informing about the action, aut name and used squishserver

Reply:
Fixed with [54eb6a5](https://github.com/CyberAlpaca/squish-api-python-extension/pull/18/commits/54eb6a5e23f52236f25c57f804e5eb38d6e58c25)